### PR TITLE
Update javadoc of @Since and @WithPlugins

### PR DESCRIPTION
**Pull request by olivergondza**

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
:+1:

File: src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java [hunk 1]
```diff
@@ -31,25 +31,24 @@
...
  * <p/>
  * One can specify a specific minimum version after the plugin name with a suffixed '@'.
  * <p/>
- * Example: @WithPlugin("subversion@1.54")
- * <p/>
- * The annotation guarantees that the plugin is installed in required or later version. If required version is not
- * available in update center, the test will fail. If the plugin is already installed but not in correct version then
+ * Example: <tt>@WithPlugin("subversion@1.54")</tt>
+ *
+ * The letter example declares that running the with older version is pointless, typically because of missing feature.
```
Comment (uhafner) [inline src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java]: latter example

Comment (uhafner) [inline src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java]: that running [the test]

Comment (olivergondza) [inline src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java]: Thanks for catching that!

**Pull request merged**